<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_CoverPreferences</class>
 <widget class="QWidget" name="GUI_CoverPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>512</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QListWidget" name="lv_cover_searchers">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_remove">
        <property name="text">
         <string notr="true">&lt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string notr="true">&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Inactive</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Active</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="lv_cover_searchers_inactive">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btn_up">
          <property name="text">
           <string notr="true">Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_down">
          <property name="text">
           <string notr="true">Down</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_delete_album_covers">
       <property name="toolTip">
        <string>Delete all covers from the database</string>
       </property>
       <property name="text">
        <string>Clear cache</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete_files">
       <property name="toolTip">
        <string>Delete all covers from the Sayonara directory</string>
       </property>
       <property name="text">
        <string>Delete files</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="cb_save_to_library">
     <property name="toolTip">
      <string>Save found covers into the library directory where the audio files are located</string>
     </property>
     <property name="text">
      <string>Save found covers into library</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="cb_fetch_from_www">
     <property name="text">
      <string>Fetch missing covers from the internet</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cb_save_to_db">
     <property name="toolTip">
      <string>Saving covers to the database leads to significantly faster access but results in a bigger database</string>
     </property>
     <property name="text">
      <string>Save found covers to database</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="cb_save_to_sayonara_dir">
     <property name="text">
      <string>Save found covers into Sayonara directory</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lab_cover_template">
       <property name="text">
        <string>Cover name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_cover_template">
       <property name="toolTip">
        <string>Name of cover file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_template_error">
       <property name="text">
        <string notr="true">Invalid chars</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_save_to_db</tabstop>
  <tabstop>cb_fetch_from_www</tabstop>
  <tabstop>lv_cover_searchers_inactive</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>lv_cover_searchers</tabstop>
  <tabstop>btn_remove</tabstop>
  <tabstop>btn_up</tabstop>
  <tabstop>btn_down</tabstop>
  <tabstop>cb_save_to_sayonara_dir</tabstop>
  <tabstop>cb_save_to_library</tabstop>
  <tabstop>le_cover_template</tabstop>
  <tabstop>btn_delete_album_covers</tabstop>
  <tabstop>btn_delete_files</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
